doc/admin/deployment.md

```diff
@@ -35,156 +35,6 @@ on boot if start-up scripts are registered with systemd.
-required directories manually. To do this execute the following commands.
-
-`daos_server`:
-```bash
-$ mkdir /var/run/daos_server
-$ chmod 0755 /var/run/daos_server
-$ chown user:user /var/run/daos_server (where user is the user you
-    will run daos_server as)
-```
-`daos_agent`:
-```bash
-$ mkdir /var/run/daos_agent
-$ chmod 0755 /var/run/daos_agent
-$ chown user:user /var/run/daos_agent (where user is the user you
-    will run daos_agent as)
-```
-
-#### Default Directory (persistent)
-
-If the server hosting `daos_server` or `daos_agent` will be rebooted often,
-systemd provides a persistent mechanism for creating the required
-directories called `tmpfiles.d`. This mechanism will be required every
-time the system is provisioned and requires a reboot to take effect.
-
-To tell systemd to create the necessary directories for DAOS:
-
--   Copy the file `utils/systemd/daosfiles.conf` to `/etc/tmpfiles.d`
-
--   Modify the copied file to change the user and group fields
-    (currently `daos`) to the user daos will be run as
-
--   Reboot the system, and the directories will be created automatically
-    on all subsequent reboots.
-
-### Elevated Privileges
-
-DAOS employs a privileged helper binary (`daos_admin`) to perform tasks
-that require elevated privileges on behalf of `daos_server`.
-
-#### Privileged Helper Configuration
-
-When DAOS is installed from RPM, the `daos_admin` helper is automatically installed
-to the correct location with the correct permissions. The RPM creates a "daos_admins"
-system group and configures permissions such that `daos_admin` may only be invoked
-from `daos_server`.
-
-For non-RPM installations, there are two supported scenarios:
-
-1. `daos_server` is run as root, which means that `daos_admin` is also invoked as root,
-and therefore no additional setup is necessary
-2. `daos_server` is run as a non-root user, which means that `daos_admin` must be
-manually installed and configured
-
-The steps to enable the second scenario are as follows (steps are assumed to be
-running out of a DAOS source tree which may be on a NFS share):
-
-```bash
-$ chmod -x $SL_PREFIX/bin/daos_admin # prevent this copy from being executed
-$ sudo cp $SL_PREFIX/bin/daos_admin /usr/bin/daos_admin
-$ sudo chmod 4755 /usr/bin/daos_admin # make this copy setuid root
-$ sudo mkdir -p /usr/share/daos/control # create symlinks to SPDK scripts
-$ sudo ln -sf $SL_PREFIX/share/daos/control/setup_spdk.sh \
-           /usr/share/daos/control
-$ sudo mkdir -p /usr/share/spdk/scripts
-$ sudo ln -sf $SL_PREFIX/share/spdk/scripts/setup.sh \
-           /usr/share/spdk/scripts
-$ sudo ln -sf $SL_PREFIX/share/spdk/scripts/common.sh \
-           /usr/share/spdk/scripts
-$ sudo ln -s $SL_PREFIX/include \
-           /usr/share/spdk/include
-```
-
-!!! note
-    The RPM installation is preferred for production scenarios. Manual
-    installation is most appropriate for development and predeployment
-    proof-of-concept scenarios.
-
```

Comment:
Wow.  All of this is no longer needed?  That is awesome!

Reply:
If you install from RPM, that's correct. If you're running a developer build you still have to set things up manually. Those instructions just got moved to a new pre-flight document in beab74f.